content/_data/changelogs/weekly.yml

```diff
@@ -17990,6 +17990,174 @@
   # pull: 7263 (PR title: Update dependency node to v16.18.0)
   # pull: 7264 (PR title: Address `set-output` deprecation)
 
+  - version: '2.375'
+    date: 2022-10-25
+    changes:
+
+      - &1
```

Comment:
What are those keywords for?

Reply:
Hi @NotMyFault I was copying over the content that was originally in the automated changelog. In that, the entries were composed as follows

` - type: TODO
    category: developer
    pull: 7274
    authors:
    - dependabot[bot]
    pr_title: Upgrade Spring Security from 5.7.3 to 5.7.4
    message: |-
      TODO fixup changelog
      Upgrade Spring Security from 5.7.3 to 5.7.4. Spring Security 5.7.4 includes [dependency upgrades and minor fixes](https://github.com/spring-projects/spring-security/releases/tag/5.7.4).
  - &1
    type: major rfe
    category: major rfe
    pull: 6912
    authors:
    - janfaracik
    - timja
    - basil
    pr_title: Improve breadcrumb bar accessibility
    message: |-
      TODO fixup changelog
      N/A
  - &2`

There were 1, 2 & 3 used, and since they appear to be major rfe's, I didn't want to exclude anything I wasn't sure of. Should those be removed, or were you referring to something else, and if so, would you be able to clarify?

Thanks!

Reply:
he means why are the yaml anchors there and not included directly &1 and then *1 etc.

occasionally the yaml library does that when generating, it won't effect the rendered version but we would normally edit them out.

Reply:
> he means why are the yaml anchors there and not included directly &1 and then *1 etc.

Yeah,  the change doesn't build, because the anchors don't point to valid locations. I took care of it in https://github.com/jenkins-infra/jenkins.io/pull/5631